linux-build.proj

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0"
+         DefaultTargets="Build"
+         xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <RHostName Condition=" '$(RHostName)'=='' ">Microsoft.R.Host</RHostName>
+    <RootDirectory Condition=" '$(RootDirectory)'=='' ">$(MSBuildThisFileDirectory)..\..\..\</RootDirectory>
```

Comment:
It's weird that we have a project file here that makes assumption about the filesystem layout outside of what's in this repository. Effectively, this line makes this project file a build artifact for the larger RTVS build, rather than specifically for the host - if you check out just R-Host and try to build this, it'll fail. So it should probably go into the RTVS repo.